app/containers/FarmingPage/reducer.js

```diff
@@ -123,63 +122,72 @@ const farmingPageReducer = (state = initialState, action) =>
       case DEFAULT_ACTION:
         break;
       case CHANGE_FARMING_CONTENT:
-        let id = `${action.payload.symbol0}-${action.payload.symbol1}`
-        let current = draft.contents.findIndex(obj => obj.deposit === id)
+        const id = `${action.payload.symbol0}-${action.payload.symbol1}`;
+        let current = draft.contents.findIndex(obj => obj.deposit === id);
         if (current >= 0) {
           draft.contents[current].totalLiquidity =
             parseInt(action.payload.reserves1) +
             parseInt(action.payload.reserves0);
         }
         break;
       case CHANGE_FARMING_CONTENT_TOKEN:
-        let token = action.payload.deposit
-        current = draft.contents.findIndex(obj => obj.deposit === token)
+        const token = action.payload.deposit;
+        current = draft.contents.findIndex(obj => obj.deposit === token);
         if (current >= 0) {
-          draft.contents[current].RGPEarned = parseInt(action.payload.tokenEarned).toFixed(4)
-          draft.contents[current].inflationPerDay = action.payload.inflationPerDay
-          draft.contents[current].tokenPrice = action.payload.tokenPrice
-          draft.contents[current].totalVolumePerPool = action.payload.totalVolumePerPool
-          draft.contents[current].ARYValue = action.payload.ARYValue
-          draft.contents[current].availableToken = action.payload.availableToken
-          draft.contents[current].tokensStaked[0] = action.payload.tokenStakedVal
-          draft.contents[current].tokensStaked[1] = action.payload.tokenEarnedVal
+          draft.contents[current].RGPEarned = parseInt(
+            action.payload.tokenEarned,
+          ).toFixed(4);
+          draft.contents[current].inflationPerDay =
+            action.payload.inflationPerDay;
+          draft.contents[current].tokenPrice = action.payload.tokenPrice;
+          draft.contents[current].totalVolumePerPool =
+            action.payload.totalVolumePerPool;
+          draft.contents[current].ARYValue = action.payload.ARYValue;
+          draft.contents[current].availableToken =
+            action.payload.availableToken;
+          draft.contents[current].tokensStaked[0] =
+            action.payload.tokenStakedVal;
+          draft.contents[current].tokensStaked[1] =
+            action.payload.tokenEarnedVal;
         }
-        break
+        break;
       case CHANGE_RGP_FARMING_FEE:
-        let farmingFee = action.payload.fee
-        draft.contents.forEach(obj => obj.farmingFee = farmingFee
-        )
-        break
+        const farmingFee = action.payload.fee;
+        draft.contents.forEach(obj => (obj.farmingFee = farmingFee));
```

Comment:
When you commit, use the -n flag, so that we can have less files to go through, some of this changes are not necessary and they make it harder for the person reviewing to actually do his work.